bundles/org.openhab.binding.mercedesme/src/main/resources/OH-INF/thing/eco-channel-types.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="mercedesme"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+	<channel-type id="accel">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Acceleration Score</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="coasting">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Coasting Score</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="constant">
+		<item-type>Number:Dimensionless</item-type>
+		<label>Constant Score</label>
+		<state readOnly="true"/>
+	</channel-type>
+	<channel-type id="bonus">
+		<item-type>Number:Length</item-type>
+		<label>Bonus Range</label>
+		<state pattern="%d km" readOnly="true"/>
```

Comment:
```suggestion
		<state pattern="%d %unit%" readOnly="true"/>
```

Doublecheck: when updating the channel, the default is m, make sure it is converted properly.

Reply:
Very good hint. Forgot to add channel to `MetaDataAdjuster`.